### ISSUE TITLE
Problem ---> Plugin Manager Window: Enter to add Plugin doesn't work when PMW is called via hotkey or started via the Menus

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#1097]
Hi,

I have this hotkey-bind for the Plugin Manager Window:

F5: ra.openPluginManager

But when I use it, I can't add a plugin via _**Enter**_.
When I use the right-click Menu, it **does** work with _**Enter**._

Any idea why that is?

Thanks,
tL.

**Comment by Lykdraft** [comment]
Aeh, actually I also can't add a plugin via double click. So basically when using the hotkey combo to call up the plugin manager window there's no way to add a plugin. 

So is there a mistake in the bind or is this more on the bug-site?

Thanks,
tL.

**Comment by kmatheussen** [comment]
I see that this current behavior is very confusing. You are not supposed to be able to add a plugin by opening the plugin manager this way. Will fix. Don't close this ticket.

As a workaround you can do this:
```
F12 CTRL_R : ra.evalScheme "(pmg-start (ra:create-new-instrument-conf) (lambda (descr) (create-instrument (ra:create-new-instrument-conf) descr)))"
```

**Comment by kmatheussen** [comment]
Fixed in 5.4.4
